python/lsst/ts/xml/data/sal_interfaces/LEDProjector/LEDProjector_Events.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>
+    LEDBasicState_Unknown=0,
+    LEDBasicState_Off=1,
+    LEDBasicState_On=2,
+    LEDBasicState_UnexpectedlyOn=3,
+    LEDBasicState_UnexpectedlyOff=4,
+    LEDBasicState_Error=5,
+    LEDBasicState_LabjackError=6,
+    LEDBasicState_LabjackConnected=7,
+    LEDBasicState_LabjackDisconnected=8
+  </Enumeration>
+  <SALEvent>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_logevent_ledState</EFDB_Topic>
```

Comment:
Would this publish as a dictionary? Or would you feed it an ID to get state?

Reply:
I was thinking it would be published on state change. In addition I was thinking it would also publish labjack status change.
So if the labjack gets connected, it would be something like {0, "Labjack", 7}

Reply:
Works for me.